ee/codegen/src/__test__/nodes/tool-calling-node.test.ts

```diff
@@ -715,5 +715,56 @@ describe("ToolCallingNode", () => {
       node.getNodeFile().write(writer);
       expect(await writer.toStringFormatted()).toMatchSnapshot();
     });
+
+    it("should handle invalid jinja blocks with missing template field", async () => {
```

Comment:
Run prettier:

Checking formatting...
[warn] src/__test__/nodes/tool-calling-node.test.ts
[warn] Code style issues found in the above file. Forgot to run Prettier?